mariadb/repo.sls

```diff
@@ -1,3 +1,4 @@
+{%- set os_family = salt['grains.get']('os_family').lower() %}
 {%- set lsb_codename = salt['grains.get']('lsb_distrib_codename') %}
 {%- set stable_version = '10.1' %}
```

Comment:
ah yea,  when hacking my own extension I noticed that one too. The site still lists 10.1 as a GA release so I did not dare to touch it :see_no_evil: